Source/WebKit/UIProcess/win/AutomationClientWin.cpp

```diff
@@ -42,22 +44,43 @@ AutomationSessionClient::AutomationSessionClient(const String& sessionIdentifier
     m_sessionIdentifier = sessionIdentifier;
 }
 
-void AutomationSessionClient::requestNewPageWithOptions(WebKit::WebAutomationSession& session, API::AutomationSessionBrowsingContextOptions, CompletionHandler<void(WebKit::WebPageProxy*)>&& completionHandler)
+AutomationSessionClient::~AutomationSessionClient()
 {
-    bool found = false;
-    auto processPool = session.protectedProcessPool();
-    if (processPool && processPool->processes().size()) {
-        // Set controlledByAutomation flag to all pages in all processes
-        processPool->setPagesControlledByAutomation(true);
-        // The default page for automation target is the first page of the first process
-        auto& processProxy = processPool->processes()[0];
-        if (processProxy->pageCount()) {
-            found = true;
-            completionHandler(processProxy->pages()[0].ptr());
-        }
-    }
-    if (!found)
-        completionHandler(nullptr);
+    for (auto webView : m_webViews)
+        ::SendMessage(webView.key, WM_DESTROY, 0, 0);
```

Comment:
It strange that we have to send native message here even though we have `WebView` object.
I think we should call `webView.close()` or just destroy the object becuase `WebView` class wraps a native window.

Reply:
Indeed. I will check.